api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsCurrentlyOnArtWithoutTbScreeningCohortQueries.java

```diff
@@ -0,0 +1,580 @@
+package org.openmrs.module.eptsreports.reporting.library.cohorts;
+
+import org.apache.commons.text.StringSubstitutor;
+import org.openmrs.Location;
+import org.openmrs.module.eptsreports.metadata.HivMetadata;
+import org.openmrs.module.eptsreports.reporting.library.queries.CommonQueries;
+import org.openmrs.module.eptsreports.reporting.utils.EptsReportUtils;
+import org.openmrs.module.reporting.cohort.definition.CohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.CompositionCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.SqlCohortDefinition;
+import org.openmrs.module.reporting.data.DataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.SqlPatientDataDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class ListOfPatientsCurrentlyOnArtWithoutTbScreeningCohortQueries {
+
+  @Autowired private TxCurrCohortQueries txCurrCohortQueries;
+
+  @Autowired private TXTBCohortQueries txtbCohortQueries;
+
+  @Autowired private CommonQueries commonQueries;
+
```

Comment:
Can all those members be autowired once through the constructor?

Reply:
Ok